public/styles/components/_link.scss

```diff
@@ -84,5 +84,12 @@ a {
         mask: url('../../assets/images/icon-envelope.svg') no-repeat 50% 50%;
       }
     }
+
+    &.bsds-link-download-button {
```

Comment:
let's swap 'button' for 'cta' here. we try to reserve link & button for their corresponding elements, and use cta when we have a link that is visually styled like a button.

looks like bsds-link-cta is winning specificity-wise for the icon.

Reply:
I see what you're saying. I didn't catch that there is no "icon-arrow-down" in this folder, and I had the wrong classname on the component. Is that icon something I export from Zeplin or does it exist somewhere else?
<img width="790" alt="Screenshot 2023-11-07 at 4 32 48 PM" src="https://github.com/bos-sci/anatomy-react/assets/96078669/3e3cda56-2144-4797-9e2f-cb5c0934f3b2">